chain/block.go

```diff
@@ -236,7 +236,7 @@ func (b *StatefulBlock) populateTxs(ctx context.Context) error {
 
 		// Verify signature async
 		if b.vm.GetVerifyAuth() {
-			txDigest, err := tx.Digest()
+			txDigest, err := tx.UnsignedBytes()
```

Comment:
Can we rename `txDigest` to `unsignedTxBytes` ?

Reply:
done